src/EditorFeatures/Core/Workspaces/AbstractTextBufferVisibilityTracker.cs

```diff
@@ -142,7 +142,7 @@ protected sealed class VisibleTrackerData : IDisposable
             /// The callbacks that want to be notified when our <see cref="TextViews"/> change visibility.  Stored as an
             /// <see cref="ImmutableHashSet{T}"/> so we can enumerate it safely without it changing underneath us.
             /// </summary>
-            public ImmutableHashSet<Action> Callbacks { get; private set; } = ImmutableHashSet<Action>.Empty;
```

Comment:
>ImmutableHashSet<Action>.Empty; [](http://example.com/codeflow?start=78&length=31)

Is ImmutableDictionary.Empty covered by this analyzer too? 

If so, was curious the usage in src\Features\Core\Portable\EditAndContinue\ActiveStatementsMap.cs wasn't changed.

Reply:
Should be. I'll check.